eventuate-plugins-gradle-publish/build.gradle

```diff
@@ -5,7 +5,8 @@ repositories {
   }
 }
 
-dependencies {
-  implementation  "io.github.gradle-nexus:publish-plugin:1.0.0"
+apply plugin: 'java'
```

Comment:
without line 8. I got the error mentioned in https://github.com/eventuate-foundation/eventuate-plugins-gradle/issues/14